Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -49,14 +49,17 @@ Widget::Widget()
     REGISTER_BOOL_PROPERTY("enabled", is_enabled, set_enabled);
     REGISTER_STRING_PROPERTY("tooltip", tooltip, set_tooltip);
 
-    REGISTER_SIZE_PROPERTY("min_size", min_size, set_min_size);
-    REGISTER_SIZE_PROPERTY("max_size", max_size, set_max_size);
+    REGISTER_UI_SIZE_PROPERTY("min_size", min_size, set_min_size);
```

Comment:
These property registers should be split into a single commit if possible

Reply:
How so? They need to be changed at the same time set_min_size is changed.